jenkins-pipelines/perf-regression-throughput-shard-aware-30min-gce.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
```

Comment:
shouldn't it be gce?

Reply:
Yes. it is bedder to have GCE  default values for GCE test